python/setup.py

```diff
@@ -37,17 +37,17 @@ def get_lua_scripts():
     packages=['ciqueue', 'ciqueue._pytest'],
     install_requires=[
         'dill>=0.2.7',
-        'pytest>=2.7,<=3.5.0',
+        'pytest>=2.7,<7',
```

Comment:
The current https://pypi.org/project/pytest/ is 7.4.0, but upgrading to it is a bigger effort, so make sure we run at least on the latest 6.x